File: CS-Examples/07_DocumentOperation/AcceptOrRejectTrackedChange.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;

namespace AcceptOrRejectTrackedChange
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Create a new Document object.
			Document document = new Document();

			// Load a Word document from the specified file path.
			document.LoadFromFile(@"..\..\..\..\..\..\..\Data\AcceptOrRejectTrackedChanges.docx");

			// Get the first Section of the document.
			Section sec = document.Sections[0];

			// Get the first Paragraph of the Section.
			Paragraph para = sec.Paragraphs[0];

			// Accept all changes made in the document.
			para.Document.AcceptChanges();

			// Specify the output file name for the modified document.
			String result = "Result-AcceptOrRejectTrackedChanges.docx";

			// Save the document with accepted changes to the specified file format (Docx2013).
			document.SaveToFile(result, FileFormat.Docx2013);

			// Dispose of the Document object to free up resources.
			document.Dispose();

            //Launch the MS Word file.
            WordDocViewer(result);
        }

        private void WordDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/07_DocumentOperation/AcceptOrRejectTrackedChange.vb
Imports System.ComponentModel
Imports System.Text
Imports Spire.Doc
Imports Spire.Doc.Documents

Namespace AcceptOrRejectTrackedChange
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			'Create a new Document object.
			Dim document As New Document()

			'Load a document from a specified file path.
			document.LoadFromFile("..\..\..\..\..\..\..\Data\AcceptOrRejectTrackedChanges.docx")

			'Get the first section of the document.
			Dim sec As Section = document.Sections(0)

			'Get the first paragraph of the section.
			Dim para As Paragraph = sec.Paragraphs(0)

			'Accept all tracked changes in the paragraph's document.
			para.Document.AcceptChanges()

			'Alternatively, reject all tracked changes in the paragraph's document.
			'para.Document.RejectChanges();

			'Specify the file name for the resulting document.
			Dim result As String = "Result-AcceptOrRejectTrackedChanges.docx"

			'Save the document to the specified file path using Docx2013 format.
			document.SaveToFile(result, FileFormat.Docx2013)

			'Dispose of the document object to free up resources.
			document.Dispose()

			'Launch the MS Word file.
			WordDocViewer(result)
		End Sub

		Private Sub WordDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
